sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -0,0 +1,190 @@
+        </div>
+        <header-title-value
+          :title="reviewLabel.assessedByLabel"
+          v-if="headerDetails.assessedBy"
+          :value="headerDetails.assessedBy"
+        />
+        <div
+          class="mx-2 vertical-divider"
+          v-if="headerDetails.assessedDate"
+        ></div>
+        <header-title-value
+          :title="reviewLabel.assessedDateLabel"
+          v-if="headerDetails.assessedDate"
+          :value="dateOnlyLongString(headerDetails.assessedDate)"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  ApplicationOfferingChangeRequestHeader,
+  ClientIdType,
+  ApplicationOfferingChangeRequestStatus,
+  ReviewLabels,
+} from "@/types";
+import HeaderTitleValue from "@/components/generic/HeaderTitleValue.vue";
+import { useFormatters } from "@/composables";
+import { computed, defineComponent } from "vue";
+import { useRouter } from "vue-router";
+import {
+  AESTRoutesConst,
+  InstitutionRoutesConst,
+} from "@/constants/routes/RouteConstants";
+import { AuthService } from "@/services/AuthService";
+
+export default defineComponent({
+  components: { HeaderTitleValue },
+  props: {
+    headerDetails: {
+      type: Object,
+      required: true,
+      default: {} as ApplicationOfferingChangeRequestHeader,
+    },
+  },
+  setup(props) {
+    const router = useRouter();
+    const { dateOnlyLongString } = useFormatters();
+    const showReviewDetails = computed(
+      () =>
+        (props.headerDetails.assessedBy || props.headerDetails.updatedDate) &&
+        (props.headerDetails.status ===
+          ApplicationOfferingChangeRequestStatus.Approved ||
+          props.headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.DeclinedBySABC ||
+          props.headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.DeclinedByStudent),
+    );
+    const reviewLabel = computed((): ReviewLabels => {
+      if (
+        props.headerDetails.assessedBy &&
+        props.headerDetails.status ===
+          ApplicationOfferingChangeRequestStatus.Approved
+      ) {
+        return {
+          assessedByLabel: "Approved By",
+          assessedDateLabel: "Approved",
+        };
+      }
+      if (
+        (props.headerDetails.assessedBy &&
+          props.headerDetails.status ===
+            ApplicationOfferingChangeRequestStatus.DeclinedBySABC) ||
+        props.headerDetails.status ===
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent
+      ) {
+        return {
+          assessedByLabel: "Declined By",
+          assessedDateLabel: "Declined",
+        };
+      }
+      return {
+        assessedByLabel: "-",
+        assessedDateLabel: "-",
+      };
+    });
+
+    const navigateToInstitutionProfile = () => {
+      if (AuthService.shared.authClientType === ClientIdType.Institution) {
+        router.push({
+          name: InstitutionRoutesConst.INSTITUTION_PROFILE,
+        });
+      }
```

Comment:
Why do you need this check and logic? this component is only used in a ministry component, right?